contract/contracts/EventManager.sol

```diff
@@ -1,19 +1,17 @@
 pragma solidity ^0.8.0;
 
-contract EventManager {
+import "@openzeppelin/contracts/utils/Strings.sol";
 
-    struct Event {
-        bytes cid;
-        uint16 ticketsTotal;
-        uint256 endDate;
-    }
+import "./EventNFT.sol";
+
+contract EventManager {
 
     uint256 constant public fee = 10 ** 17;
 
-    mapping(address => Event[]) public events;
+    address[] public createdEvents;
```

Comment:
do you need this array just to receive size of it?

Reply:
Not only: My thoughts was to allow external users to easily retrieve the whole list of created events. To browse in our FE for example.

Reply:
but it would be done via the graph for FE